test/suites/edgex-no-sec/main_test.go

```diff
@@ -69,10 +69,14 @@ func setup() (teardown func(), err error) {
 		utils.SnapDumpLogs(nil, start, ekuiperSnap)
 		utils.SnapDumpLogs(nil, start, ascSnap)
 
-		utils.SnapRemove(nil, platformSnap)
-		utils.SnapRemove(nil, deviceVirtualSnap)
-		utils.SnapRemove(nil, ekuiperSnap)
-		utils.SnapRemove(nil, ascSnap)
+		if utils.SkipTeardownRemoval {
+			log.Println("[SKIP] Removal of snaps in teardown is skipped.")
+		} else {
```

Comment:
```suggestion
                log.Println("Removing installed snap:", utils.SkipTeardownRemoval)
		if !utils.SkipTeardownRemoval {
			...
```

Reply:
As `SkipTeardownRemoval` is set to true by default, the current output 'Removing installed snap: true' is inaccurate. To provide a more precise output, we could use one of the following alternatives:
```
log.Println("Removing installed snap:", !utils.SkipTeardownRemoval)
# Removing installed snap: false
```
```
log.Println("Skipping removal of snaps:", utils.SkipTeardownRemoval)
# Skipping removal of snaps: true
```

Reply:
Right, that is completely false, not just inaccurate. 

> `log.Println("Removing installed snap:", !utils.SkipTeardownRemoval)`

Looks better because it prints what is about to happen next.